docs/docs/icicle/golang-bindings.md

```diff
@@ -33,28 +33,30 @@ go get github.com/ingonyama-zk/icicle@<commit_id>
 
 To build the shared libraries you can run this script:
 
+```bash
+./build.sh <curve> [-cuda_version=<version>] [-g2] [-ecntt] [-devmode]
```

Comment:
<curve> was moved to a flag and `-field=<field>` and `field-ext` were added as well. Usage can be displayed with the flag `-help`

Reply:
fixed